noxfile.py

```diff
@@ -41,7 +41,7 @@ def venv_cached(session):
 
     """
     result = False
-    yml = Path(f"requirements/ci/py{PY_VER.replace('.', '')}.yml")
+    yml = Path(f"requirements/ci/py{session.python.replace('.', '')}.yml")
```

Comment:
See [nox documentation](https://nox.thea.codes/en/stable/config.html#nox.sessions.Session.python) for further details.